client/services/code_preprocessors/PythonCodePreprocessorService.js

```diff
@@ -54,43 +93,66 @@ tie.factory('PythonCodePreprocessorService', [
       correctnessTests.forEach(function(test, index) {
         testCode += '\n' + [
           VARNAME_TEST_RESULTS + '.append(',
-          '    run_test(' + jsonVariableToPython(test.getInput()) + '))'
+          '    run_correctness_test(' + jsonVariableToPython(test.getInput()) + '))'
         ].join('\n');
       });
 
       return testCode;
-    }
+    };
 
-    return {
-      // Wraps a WRAPPER_CLASS_NAME class around the series of functions in a
-      // given code snippet.
-      _wrapCodeIntoClass: function(code) {
-        var codeLines = code.trim().split('\n');
+    var generatePerformanceTestCode = function(performanceTests) {
+      performanceTests.forEach(function(test, index) {
+        var qualifiedEvaluationFunctionName = (
+          WRAPPER_CLASS_NAME + '().' + test.getEvaluationFunction());
+        var qualifiedTransformationFunctionName = (
+          WRAPPER_CLASS_NAME + '().' + test.getTransformationFunction());
+        // TODO(eyurko): Make this work for other runtimes.
+        // TODO(eyurko): 30 seems very scientific. Maybe stop hardcoding?
```

Comment:
Unclear exactly what needs to be done here. Could you rewrite to specify what exactly you want to do (or want someone else to do) instead?